### PR TITLE
Fix app catalog name reference in indexing step

### PR DESCRIPTION
**Pull request by sslavic**

### CODE REVIEW

**Comment by kopiczko** [comment]
Please also follow steps from releases section: https://github.com/giantswarm/architect-orb#releases

We need to create 0.1.1 release here.

**Comment by sslavic** [comment]
@kopiczko 0.1.1 release has been prepared. PTAL

**Comment by sslavic** [comment]
Btw, got one more confirmation that the fix is needed, EFK helm chart publishing indexed the package with broken URL, see https://github.com/giantswarm/giantswarm-incubator-test-catalog/blob/master/index.yaml

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.1] - 2019-06-14
+
+### Changed
```
Comment (kopiczko) [inline CHANGELOG.md]:
```suggestion
### Fixed
```

Comment (sslavic) [reply CHANGELOG.md]: Suggestion applied and agree with this convention.

To explain myself, https://keepachangelog.com/en/1.0.0/ examples are misleading, top ones coupled fixes under `Changed` and initially I just saw those examples; now that you suggested change I checked once more and found there are examples further below which have fixes under dedicated `Fixed` paragraph in changelog.

Comment (sslavic) [reply CHANGELOG.md]: @kopiczko PTAL

Comment (kopiczko) [reply CHANGELOG.md]: Yeah we have to get used to that.

Comment (kopiczko) [reply CHANGELOG.md]: Still "best" standard I have found. Feel free to suggest something more structured if you have something.

**Pull request merged**